modules/extension/app-schema/app-schema/pom.xml

```diff
@@ -188,6 +188,18 @@
             <version>3.2.0-1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-solr</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+		      <groupId>org.slf4j</groupId>
+		      <artifactId>slf4j-api</artifactId>
+		      <version>1.7.7</version>
+		      <scope>test</scope>
+		    </dependency>
```

Comment:
The Apache Solr integration tests should be moved to ``gt-solr`` module.

Reply:
Done